packages/webpack-cli/lib/utils/get-package-manager.js

```diff
@@ -10,9 +10,19 @@ const { sync } = require('execa');
  * @returns {String} - The package manager name
  */
 function getPackageManager() {
-    const hasLocalYarn = fs.existsSync(path.resolve(process.cwd(), 'yarn.lock'));
     const hasLocalNpm = fs.existsSync(path.resolve(process.cwd(), 'package-lock.json'));
-    const hasLocalPnpm = fs.existsSync(path.resolve(process.cwd(), 'pnpm-lock.yaml'));
+
+    let hasLocalYarn = false;
+
+    if (!hasLocalNpm) {
+        hasLocalYarn = fs.existsSync(path.resolve(process.cwd(), 'yarn.lock'));
+    }
+
+    let hasLocalPnpm = false;
+
+    if (!hasLocalYarn) {
+        hasLocalPnpm = fs.existsSync(path.resolve(process.cwd(), 'pnpm-lock.yaml'));
```

Comment:
```suggestion
        hasLocalPnpm = fs.existsSync(path.resolve('pnpm-lock.yaml'));
```

Reply:
`process.cwd()` is required. else it will resolve to `/home/snitin315/Projects/webpack-cli/package-lock.json` when it should be `/home/snitin315/Projects/webpack-cli/packages/webpack-cli/lib/utils/__tests__/test-all-lock/package-lock.json`

Reply:
Ah yes, it is required; since we are mocking `process.cwd()`.